environs/config/config.go

```diff
@@ -531,6 +535,21 @@ func Validate(cfg, old *Config) error {
 		}
 	}
 
+	if v, ok := cfg.defined[FanConfig].(string); ok && v != "" {
```

Comment:
This should be broken out and put in the network package.

Reply:
Done